app/src/main/java/com/appdev/eateryblueandroid/ui/components/profile/AccountSummaries.kt

```diff
@@ -1,9 +1,103 @@
 package com.appdev.eateryblueandroid.ui.components.profile
 
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.unit.dp
+import com.appdev.eateryblueandroid.R
+import com.appdev.eateryblueandroid.models.Account
+import com.appdev.eateryblueandroid.models.AccountType
 import com.appdev.eateryblueandroid.ui.components.core.Text
+import com.appdev.eateryblueandroid.ui.components.core.TextStyle
 
 @Composable
-fun AccountSummaries() {
-    return Text(text = "ACCOUNT SUMMARIES")
+fun AccountSummaries(accounts: List<Account>?) {
+    val swipes = accounts?.find { it.type == AccountType.MEALPLAN }?.balance
+    val brbs = accounts?.find { it.type == AccountType.BRBS }?.balance ?: 0.0
```

Comment:
Why are BRBs the only ones that require null coalescing to 0.0?

Reply:
I thought it would be better to always display the person's BRB balance (even if they didn't have a BRB account). Just because BRB use is so common. Added a comment for this (the other values will be hidden if user does not have a corresponding account for them).